packages/fetch-router/demos/node/app/templates.ts

```diff
@@ -0,0 +1,54 @@
+import { html, type SafeHtml } from '@remix-run/html-template'
+
+import type { Post } from '../data.ts'
+import { routes } from '../routes.ts'
+import { getPostHrefParams } from './utils.ts'
+
+export function layout(body: SafeHtml | string, currentUser?: string) {
```

Comment:
Because we don't have a concept of nested routing I end up having to call `layout` on every single GET route. Maybe that's fine, but felt a little obnoxious, especially with passing in `currentUser`